x-pack/plugins/ml/public/application/timeseriesexplorer/timeseriesexplorer_page.tsx

```diff
@@ -38,13 +39,11 @@ export const TimeSeriesExplorerPage: FC<PropsWithChildren<TimeSeriesExplorerPage
   const CasesContext = cases?.ui.getCasesContext() ?? React.Fragment;
   const casesPermissions = cases?.helpers.canUseCases();
   const helpLink = docLinks.links.ml.anomalyDetection;
+
+  const timeseriesExplorerStyles = getTimeseriesExplorerStyles();
```

Comment:
Could this be done once outside the component instead of during every render?

Reply:
Done in  [0d5af5f](https://github.com/elastic/kibana/pull/195259/commits/0d5af5f5e20ed4cb007068523929eb0079e7e25a)